docs/wiki/01 - STUDY PARAMETERS SETUP/01 - Study Variables/03 - Sites.md

```diff
@@ -0,0 +1,45 @@
+# Sites
+
+## Overview Study
+
+**Sites** (preferred) or **Centers** are terms used interchangeably 
+in LORIS. **Sites** are defined in the `psc` table of the database 
+(_psc_ stands for _Project Study Center_).
+
+LORIS' default schema defines the first `psc` (`CenterID=1`) as the
+Data Coordinating Center (DCC).  This site is generally used for
+dummy data and it is assumed by the codebase to store exclusively 
+non-study data. Therefore, it is recommended not to modify or use 
+this site for registering real participants or study data.
+
+>  Note: There cannot be more than one site with the same name.
```

Comment:
This note seems random here.  Would move it under the MySQL insert and change "name" to `Name` so that it's clear which field.

Reply:
well it applies regardless of how we insert the sites into the database

Reply:
now that i think about it... it should be completely removed... its a given it's like saying there cannot be 2 projects with the same name and 2 subprojects

what do you think ?

Reply:
@ridz1208 agreed, remove it.

Reply:
what's wrong with saying there cannot be 2 projects with the same name or 2 subprojects? "Name" being unique isn't really a given. There are likely many people in the world who share your name, for instance.

Reply:
none of our entities support having duplicates... I just dont see the need to add it to every single one of our docs.